browsers/browsers.schema.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+
+  "definitions": {
+
+    "browser_statement": {
+      "type": "object",
+      "properties": {
+        "releases": {
+          "type": "object",
+          "additionalProperties": { "$ref": "#/definitions/release_statement" }
+        }
+      },
+      "required": ["releases"],
+      "additionalProperties": false
+    },
+
+    "release_statement": {
+      "type": "object",
+      "properties": {
+        "release_date": { "type": "string", "format": "date" },
+        "release_notes": { "type": "string", "format": "uri" },
+        "status": {
+          "type": "string",
+          "enum": ["retired", "current", "beta", "nightly", "esr", "planned"]
```

Comment:
I'm assuming these names work across all browser vendors? Do we need a name for browsers that are older release versions, not the latest version, but still current and in need of supporting? e.g. IE10 and IE11 .

Reply:
I marked IE10 as "retired" and IE11 as "current". Is that what you would expect, too?

```diff
+      "10": {
+        "release_date": "2012-10-26",
+        "status": "retired"
+      },
+      "11": {
+        "release_date": "2013-10-17",
+        "status": "current"
+      }
```

Reply:
Well, it is "the most current version", so if that's what you want it to mean, then we're all good. But it certainly isn't a "current" browser to my mind ;-)